### DISCUSSION TITLE
Naming csv header

### DISCUSSION THREAD

**Discussion by horsto** [bonsai-rx/bonsai#987] [Q&A] [answered]
I am trying to rename zipped values so that they carry explicit descriptions in the header of a .csv file that I am writing after the output of the workflow (see screenshot), instead of just being named `Item1. ...`. I am trying to use an `Expression Transform` for this, but this runs into the exception "Build Error: Expression expected". What is the correct way of doing this?  

<img width="1013" alt="Screenshot 2022-07-26 at 13 50 42" src="https://user-images.githubusercontent.com/7548928/181076732-f667bd81-4edc-48b1-bc4e-3a4dbebf256f.png">

**Comment by bruno-f-cruz** [comment, marked as answer]
Try to remove the last comma.

**Reply by horsto** [reply]
That did the trick, thanks!

**Comment by horsto** [comment]
Random follow-up question: The `Zip`: Is that limited in how many values it can combine at once? I noticed that at a certain number of inputs it changes the format from `System.Tuple <float, float ,....>` (in my case) to something list / float.

**Reply by bruno-f-cruz** [reply]
`Tuples` are currently limited to 7 elements.

**Reply by horsto** [reply]
Thanks. Any spontaneous trick / idea on how to deal with combining > 10 values into one output for the `csvwriter` ? (python transform?)

**Reply by bruno-f-cruz** [reply]
Tuples of tuples should do the trick.
